pkg/kubernetes/common.go

```diff
@@ -51,6 +51,9 @@ func GetDaprHelmChartName(helmConf *helm.Configuration) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	if len(releases) == 0 {
+		return "", errors.New("dapr is not installed in your helm")
```

Comment:
nit: Just thinking of better error message here -
"Could not find release name dapr in the your helm releases" @shubham1172 WDYT?

Reply:
Yes, we should inform users transparently. +1